core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1782,6 +1782,9 @@ write_files:
           - --oidc-groups-claim={{.Experimental.Oidc.GroupsClaim}}
           {{ end -}}
           {{ end -}}
+          {{if .Experimental.EncryptionAtRest.Enabled}}
```

Comment:
We have started to keep the `experimental` key at distance so I'm now rather ok with something like:

```yaml
kubernetes:
  encryptionAtRest:
    enabled: true
```

Reply:
Addressed in a follow-up commit